lib/datasource/artifactory/readme.md

```diff
@@ -0,0 +1,3 @@
+This datasource returns releases from given custom `registryUrl`(s).
```

Comment:
```suggestion
This datasource returns releases from given custom `registryUrl`(s).
Artifactory is the recommended registry for Conan packages.
```

Maybe we should put the string `Conan` in our docs, as people won't be able to find anything about Conan when searching through the search function later on... 😄 

I did a search for the string `conan` in the PR code, and I don't find any mention of this string.

Reply:
Good point. Now, conan is mentioned on the readme.

Reply:
Great! 😄